src/main/java/frc/robot/commands/ActiveSetShooter.java

```diff
@@ -31,29 +39,45 @@ public ActiveSetShooter(Shooter shooter, ShooterPivot shootPivot, Supplier<Shoot
     // Called when the command is initially scheduled.
     @Override
     public void initialize() {
-        // start the feeder motor and timer to back the NOTE a bit
-        m_shooter.setFeederSpeed(Shooter.BACKUP_FEED_SPEED);
-        m_shooter.setShooterSpeeds(Shooter.BACKUP_SHOOTER_SPEED, Shooter.BACKUP_SHOOTER_SPEED);
-        m_backupTimer.restart();
+        m_initialRotations = m_shooter.getFeederRotations();
+
+        m_timer.restart();
+        m_state = State.WAIT_FOR_PIVOT;
     }
 
     // Called every time the scheduler runs while the command is scheduled.
     @Override
     public void execute() {
+        // always adjust the pivot
         Shooter.ShooterValues shootValues = m_valueSupplier.get();
         m_shooterPivot.setAngle(shootValues.shootAngle);
 
-        if (m_backupTimer.hasElapsed(Shooter.BACKUP_FEED_TIME))
-        {
+        if (m_state == State.WAIT_FOR_PIVOT && m_timer.hasElapsed(PIVOT_WAIT_TIME)) {
```

Comment:
This is still using the timer. Shouldn't we be checking if the pivot achieved the requested goal? Instead of using the m_timer.hasElapsed(), should we use m_shooterPivot.angleWithinTolerance()?

Reply:
I will add this into the new branch, OK?